src/pages/join.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState } from "react"; 
 import type { NextPage } from "next";
-import styled from "@emotion/styled"
+import styled from "@emotion/styled" 
```

Comment:
p2: prettier설정을 확인해보시는게 좋을 거 같네요! 세미콜론이 빠졌네요.
```suggestion
import styled from "@emotion/styled" ;
```

Reply:
넵 확인하겠습니다